pkg/cmd/publish-provisional-artifacts/main.go

```diff
@@ -58,23 +54,18 @@ func makeS3() (s3I, error) {
 	return s3.New(sess), nil
 }
 
-var osVersionRe = regexp.MustCompile(`\d+(\.\d+)*-`)
-
 var isReleaseF = flag.Bool("release", false, "build in release mode instead of bleeding-edge mode")
 var destBucket = flag.String("bucket", "", "override default bucket")
 var doProvisionalF = flag.Bool("provisional", false, "publish provisional binaries")
 var doBlessF = flag.Bool("bless", false, "bless provisional binaries")
 
 var (
-	noCache = "no-cache"
```

Comment:
Why only move `noCache` from here and not from `publish-artifacts`?

Reply:
Oops! Done!